docs/source/onnxruntime/usage_guides/gpu.mdx

```diff
@@ -0,0 +1,225 @@
+# Use Nvidia GPU for inference through Optimum-onnxruntime
+
+Although ONNX Runtime defaults to use CPU-only for inference, it is possible to place supported operations on an Nvidia GPU, while leaving unsupported ones (if remaining) on CPU. In most cases, all costly operations can be placed on GPU, allowing to leverage its compute power.
+
+ONNX Runtime provides an integration with two execution providers to make use of an Nvidia GPU, namely `CUDAExecutionProvider` and `TensorrtExecutionProvider`.
+
+In case you are using a vanilla model using floating-point arithmetic, both execution providers can be used. TensorRT generally has the best runtime performances, although it comes with its own caveats, notably limitations for inference with inputs having dynamic shape.
+
+When deploying a quantized model to a GPU, `CUDAExecutionProvider` can not be used, and only models using static quantization can be executed on `TensorrtExecutionProvider`.
+
+## CUDAExecutionProvider
+
+### Installation
+
+Provided the CUDA and cuDNN [requirements](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements) are satisfied, the install is then straightforward with `pip install optimum[onnxruntime-gpu]`. Beforehand, to avoid conflicts between `onnxruntime` and `onnxruntime-gpu`, make sure the package `onnxruntime` is not installed by running `pip uninstall onnxruntime`.
+
+### Checking the installation is successful
+
+Before going further, run the following sample code to check whether the install was successful:
+
+```python
+from optimum.onnxruntime import ORTModelForSequenceClassification
+from transformers import AutoTokenizer
+
+ort_model = ORTModelForSequenceClassification.from_pretrained(
+    "philschmid/tiny-bert-sst2-distilled",
+    from_transformers=True,
+    provider="CUDAExecutionProvider",
+)
+print("providers:", ort_model.providers)
+
+tokenizer = AutoTokenizer.from_pretrained("philschmid/tiny-bert-sst2-distilled")
+inp = tokenizer("expectations were low, actual enjoyment was high", return_tensors="pt", padding=True)
+
+result = ort_model(**inp)
+print(result)
+```
+
+In case the printed result is `providers: ['CUDAExecutionProvider', 'CPUExecutionProvider']`, congratulations, the installation is successful!
+
+In case the error `ValueError: Asked to use CUDAExecutionProvider as an ONNX Runtime execution provider, but the available execution providers are ['CPUExecutionProvider'].` is raised, something is off in your installation.
```

Comment:
```suggestion
If you encounter the following error:

```python
ValueError: Asked to use CUDAExecutionProvider as an ONNX Runtime execution provider, but the available execution providers are ['CPUExecutionProvider'].
```

then something is wrong with the ONNX Runtime installation.
```